### DISCUSSION TITLE
Error in Docker compose [failed to solve: process "/bin/sh -c ./gradlew --no-daemon --info]

### DISCUSSION THREAD

**Discussion by Taichi415** [apache/polaris#494] [Q&A]
Hi I am following the Quick Start document of Apache Polaris below.
https://polaris.apache.org/in-dev/unreleased/quickstart/
↓Version of Polaris
https://github.com/apache/polaris/pull/492

And facing an Error below and cant figure out how to fix this.
I asked Chatgpt for help and it says Kotlin built is failed and I need to fix it.
Is there any files or setting I need to check ??

>  => ERROR [polaris build 7/7] RUN ./gradlew --no-daemon --info  ${ECLIPSELINK_DEPS+"-PeclipseLinkDeps=$ECLIPSELINK_DEPS"} -PeclipseLink=false clean prepareDockerDist                                               253.4s 
> ------
>  > [polaris build 7/7] RUN ./gradlew --no-daemon --info  ${ECLIPSELINK_DEPS+"-PeclipseLinkDeps=$ECLIPSELINK_DEPS"} -PeclipseLink=false clean prepareDockerDist:
> 0.552   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
> 0.552                                  Dload  Upload   Total   Spent    Left  Speed
> 100   167  100   167    0     0    543      0 --:--:-- --:--:-- --:--:--   543
> 100    64  100    64    0     0     63      0  0:00:01  0:00:01 --:--:--     0
> 1.569   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
> 1.569                                  Dload  Upload   Total   Spent    Left  Speed
> 100 43583  100 43583    0     0  81311      0 --:--:-- --:--:-- --:--:-- 81463
> 2.324 Downloading https://services.gradle.org/distributions/gradle-8.11.1-bin.zip
> 5.262 .............10%.............20%.............30%.............40%.............50%.............60%.............70%.............80%.............90%.............100%
> 203.8 Initialized native services in: /home/default/.gradle/native
> 203.9 Initialized jansi services in: /home/default/.gradle/native
> 204.0
> 204.0 Welcome to Gradle 8.11.1!
> 204.0
> 204.0 Here are the highlights of this release:
> 204.0  - Parallel load and store for Configuration Cache
> 204.0  - Java compilation errors at the end of the build output
> 204.0  - Consolidated report for warnings and deprecations
> 204.0
> 204.0 For more details see https://docs.gradle.org/8.11.1/release-notes.html
> 204.0
> 204.2 To honour the JVM settings for this build a single-use Daemon process will be forked. For more on this, please refer to https://docs.gradle.org/8.11.1/userguide/gradle_daemon.html#sec:disabling_the_daemon in the Gradle documentation.
> 204.2 Starting process 'Gradle build daemon'. Working directory: /home/default/.gradle/daemon/8.11.1 Command: /usr/lib/jvm/java-21-openjdk-21.0.4.0.7-1.el9.x86_64/bin/java --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.invoke=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.prefs/java.util.prefs=ALL-UNNAMED --add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED --add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.prefs/java.util.prefs=ALL-UNNAMED --add-opens=java.base/java.nio.charset=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.util.concurrent.atomic=ALL-UNNAMED --add-opens=java.xml/javax.xml.namespace=ALL-UNNAMED -XX:MaxMetaspaceSize=768m -Xms2g -Xmx4g -Dfile.encoding=UTF-8 -Duser.country -Duser.language=en -Duser.variant -cp /home/default/.gradle/wrapper/dists/gradle-8.11.1-bin/bpt9gzteqjrbo1mjrsomdt32c/gradle-8.11.1/lib/gradle-daemon-main-8.11.1.jar -javaagent:/home/default/.gradle/wrapper/dists/gradle-8.11.1-bin/bpt9gzteqjrbo1mjrsomdt32c/gradle-8.11.1/lib/agents/gradle-instrumentation-agent-8.11.1.jar org.gradle.launcher.daemon.bootstrap.GradleDaemon 8.11.1
> 204.2 Successfully started process 'Gradle build daemon'
> 206.1 An attempt to start the daemon took 1.894 secs.
> 206.3 The client will now receive all logging from the daemon (pid: 67). The daemon log file: /home/default/.gradle/daemon/8.11.1/daemon-67.out.log
> 206.3 Daemon will be stopped at the end of the build
> 207.3 Using 8 worker leases.
> 207.3 Configuration on demand is an incubating feature.
> 208.2 Received JVM installation metadata from '/usr/lib/jvm/java-21-openjdk-21.0.4.0.7-1.el9.x86_64': {JAVA_HOME=/usr/lib/jvm/java-21-openjdk-21.0.4.0.7-1.el9.x86_64, JAVA_VERSION=21.0.4, JAVA_VENDOR=Red Hat, Inc., RUNTIME_NAME=OpenJDK Runtime Environment, RUNTIME_VERSION=21.0.4+7-LTS, VM_NAME=OpenJDK 64-Bit Server VM, VM_VERSION=21.0.4+7-LTS, VM_VENDOR=Red Hat, Inc., OS_ARCH=amd64}
> 209.4 Watching the file system is configured to be enabled if available
> 209.5 Not watching /app since the file system is not supported
> 209.5 File system watching is active
> 211.5 Starting Build
> 211.8 Generating /home/default/.gradle/caches/8.11.1/generated-gradle-jars/gradle-api-8.11.1.jar
> 232.3 Cannot use Kotlin build script compile avoidance with /home/default/.gradle/caches/8.11.1/generated-gradle-jars/gradle-api-8.11.1.jar: class org/gradle/internal/impldep/kotlinx/coroutines/scheduling/CoroutineScheduler: inline fun availableCpuPermits(): compile avoidance is not supported with public inline functions
> 235.4 Cannot use Kotlin build script compile avoidance with /home/default/.gradle/wrapper/dists/gradle-8.11.1-bin/bpt9gzteqjrbo1mjrsomdt32c/gradle-8.11.1/lib/gradle-kotlin-dsl-extensions-8.11.1.jar: class org/gradle/kotlin/dsl/GradleApiKotlinDslExtensions_1cbh1oqkvm762j10e96dawuh5Kt: inline fun domainObjectContainer(): compile avoidance is not supported with public inline functions
> 235.4 Cannot use Kotlin build script compile avoidance with /home/default/.gradle/wrapper/dists/gradle-8.11.1-bin/bpt9gzteqjrbo1mjrsomdt32c/gradle-8.11.1/lib/gradle-kotlin-dsl-8.11.1.jar: class org/gradle/kotlin/dsl/ArtifactHandlerScope: inline fun invoke(): compile avoidance is not supported with public inline functions
> 235.5 Caching disabled for Kotlin DSL script compilation (Settings/TopLevel/stage1) because:
> 235.5   Build cache is disabled
> 249.6 Caching disabled for Kotlin DSL accessors for settings 'app' because:
> 249.6   Build cache is disabled
> 249.8 Caching disabled for Kotlin DSL script compilation (Settings/TopLevel/stage2) because:
> 249.8   Build cache is disabled
> 252.6 Settings evaluated using settings file '/app/settings.gradle.kts'.
> 252.6 Using local directory build cache for the root build (location = /home/default/.gradle/caches/build-cache-1, remove unused entries = after 7 days).
> 252.9 Not watching /app/build-logic since the file system is not supported
> 252.9 [Incubating] Problems report is available at: file:///app/build/reports/problems/problems-report.html
> 252.9 Some of the file system contents retained in the virtual file system are on file systems that Gradle doesn't support watching. The relevant state was discarded to ensure changes to these locations are properly detected. You can override this by explicitly enabling file system watching.
> ------
> failed to solve: process "/bin/sh -c ./gradlew --no-daemon --info  ${ECLIPSELINK_DEPS+\"-PeclipseLinkDeps=$ECLIPSELINK_DEPS\"} -PeclipseLink=$ECLIPSELINK clean prepareDockerDist" did not complete successfully: exit code: 1

**Comment by MonkeyCanCode** [comment]
Checking

**Comment by MonkeyCanCode** [comment]
@Taichi415 just tried it locally and it still work for me. Can u upload the content of `file:///app/build/reports/problems/problems-report.htm` mentioned in the above logs:
```
252.9 [Incubating] Problems report is available at: file:///app/build/reports/problems/problems-report.html
```
